LumpNav/LumpNav.py

```diff
@@ -40,18 +40,71 @@ def __init__(self, parent = None):
     GuideletWidget.__init__(self, parent)
     
   def setup(self):
+    # Adds default configurations to Slicer.ini
+    self.addDefaultConfiguration()
+    
     GuideletWidget.setup(self)
-
+    
   def addLauncherWidgets(self):
     GuideletWidget.addLauncherWidgets(self)
 
+    # Configurations
+    self.configurations()
+    
+    # BreachWarning
+    self.breachWarningLight()  
+ 
+  # Adds default configurations to Slicer.ini
+  def addDefaultConfiguration(self):
+    settings = slicer.app.userSettings()    
+    if not settings.value(self.moduleName + '/Configurations/default/TipToSurfaceDistanceCrossHair'):

```

Comment:
Have you tried begingroup? You may also try to go to Configurations and then check childGroups().

Worst case you could check for some required value, such as add a "Name" value, which would be a human-readable name for the configuration. It would be much nicer to check the existence of that than the very specific "TipToSurfaceDistanceCrossHair".